report_paper_wkhtmltopdf_params/models/report.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Avoin.Systems
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class Report(models.Model):
+    _inherit = 'report'
+
+    def _build_wkhtmltopdf_args(self, paperformat,
+                                specific_paperformat_args=None):
+        # noinspection PyUnresolvedReferences,PyProtectedMember
```

Comment:
This is actually very interesting, when you work with PyCharm, Will remove warning messages.